pdfbox/src/main/java/org/apache/pdfbox/rendering/PageDrawer.java

```diff
@@ -1602,9 +1607,9 @@ private TransparencyGroup(PDTransparencyGroup form, boolean isSoftMask, Matrix c
             GeneralPath transformedBox = form.getBBox().transform(transform);
 
             // clip the bbox to prevent giant bboxes from consuming all memory
-            Area clip = (Area)getGraphicsState().getCurrentClippingPath().clone();
-            clip.intersect(new Area(transformedBox));
-            Rectangle2D clipRect = clip.getBounds2D();
+            Area transformed = new Area(transformedBox);
+            transformed.intersect(getGraphicsState().getCurrentClippingPath());
+            Rectangle2D clipRect = transformed.getBounds2D();

```

Comment:
Unrelated, but I noticed that the call to `clone()` is not necessary if the areas are swapped.